src/Components/Home.js

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
-import '../Styles/Home.scss';
 import PetContainer from './PetContainer';
 import Header from './Header';
+import { useQuery, gql } from '@apollo/client';
+import '../Styles/Home.scss';
+
+const GET_PETS = gql`
```

Comment:
I'm excited to start using queries, and learning how this data will flow through our app!